nstar/src/communication.rs

```diff
@@ -49,7 +65,7 @@ pub fn start_receiving_from_socket(
 ) -> Result<sync::broadcast::Sender<Response>> {
     let (response_sender, _) = sync::broadcast::channel(100);
     let sender = response_sender.clone();
-    let _ = tokio::spawn(async move {
+    task::spawn(async move {
```

Comment:
is there a difference between `task::spawn` and `tokio::spawn`?

Reply:
No. But we use the `task` module all over. The `_` binding is useless.